presto-main/src/test/java/io/prestosql/operator/scalar/date/TestExtract.java

```diff
@@ -42,20 +42,23 @@ public void teardown()
     @Test
     public void testYear()
     {
+        assertThat(assertions.expression("year(DATE '2020-05-10')")).matches("BIGINT '2020'");
```

Comment:
In other TestExtract test classes we put `EXTRACT(...` cases first within a test method, then followed by the cases covering the convenience methods. See `io.prestosql.operator.scalar.timestamptz.TestExtract` for a "canonical example". Can you rearrange the cases here?